be/src/storage/persistent_index.cpp

```diff
@@ -270,4 +717,17 @@ Status PersistentIndex::erase(size_t n, const void* keys, IndexValue* old_values
     return Status::OK();
 }
 
+Status PersistentIndex::_flush_l0() {
```

Comment:
When to flush L0 mutable index, how about the accessing call by in-progress loading?
Will it be stalled or make a snapshot of L0 to make the process can progress?

Reply:
Currently, it's all synchronized flush